### ISSUE TITLE
Mono audio causes TypeError when calling the dump_wav function

### ISSUE THREAD

**Issue by pstrepetov** [Yikai-Liao/symusic#54]
symusic version: 0.5.4

**Code from tutorial:**

```
from symusic import Score, Synthesizer, BuiltInSF3 ,dump_wav

s = Score(midi_path)

sf_path = BuiltInSF3.MuseScoreGeneral().path(download=True)
synth = Synthesizer(
    sf_path = sf_path, # the path to the soundfont
    sample_rate = 44100, # the sample rate of the output wave, 44100 is the default value
)

audio = synth.render(s, stereo=False)
dump_wav("out.wav", audio, sample_rate=44100, use_int16=True)
```

**Causes an error:**
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
Cell In[24], line 1
----> 1 dump_wav("out.wav", audio, sample_rate=44100, use_int16=True)

TypeError: dump_wav(): incompatible function arguments. The following argument types are supported:
    1. dump_wav(path: str, data: numpy.ndarray[dtype=float32, shape=(*, *), order='F'], sample_rate: int, use_int16: bool = True) -> None

Invoked with types: str, ndarray, kwargs = { sample_rate: int, use_int16: bool }
```

Everything works fine with the passed argument stereo=True

**Comment by lzqlzzq** [comment]
That' s a strange error. I checked this error only be raised when `shape` of `audio` is `(1, *)`. But with shape of `(3, *)`, `(4, *)` and even more channel, `dump_wav` can dump them correctly......
I would investigate if that problem is from `nanobind`.

**Comment by lzqlzzq** [comment]
Fixed in commit a67dd3d. You may build `symusic` from source to address the problem for now.

**Comment by pstrepetov** [comment]
Thank you!
